main.py

```diff
@@ -657,6 +658,13 @@ def batch_process(users_chunk, batch_args):
         ]
         tags.extend(repo_tags)
 
+    explicit_tags = env_config["tags"].split(",")
```

Comment:
Write above this block, there is logic to generate repo tags which is redundant. Can we create branching here so that when we specify explicit tags, there is only loop?